### PR TITLE
Sql procedures

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Create SQL Procedures entry.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Review by yangc95 — COMMENTED** [review]
@SSwiniarski Review 1️⃣ done! I also saw that some favor object-relational mapping to stored procedures. I'm wondering if this is something that can be mentioned or sourced out to for another docs entry. I also understand if this is out-of-scope in which case we can just save it entirely for another entry.

File: content/sql/concepts/procedures/procedures.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+Title: 'Procedures'
+Description: 'Procedures (also stored procedures) are blocks of SQL code that are saved in a database and can be executed repeatedly on demand.'
```
[the remark this answers is not in the document]
Comment (SSwiniarski) [reply content/sql/concepts/procedures/procedures.md:3]: I think the object-relational tangent should be its own entry, since not all DBMSs implement it.

**Comment by SSwiniarski** [comment]
@yangc95, I applied most of your suggestions. I think object-relational mapping belons in its own entry.

File: content/sql/concepts/procedures/procedures.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'MySQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+Procedures are blocks of SQL code that are saved in a database and can be executed repeatedly on demand. Also referred to as stored procedures — they can be defined with parameters to be used within the body of the procedure. They can also define parameters that return an output value.
```
Comment (caupolicandiaz) [inline content/sql/concepts/procedures/procedures.md:14]:
```suggestion
Procedures are blocks of SQL code that are saved in a database and can be executed repeatedly on demand. Also referred to as stored procedures — they can be defined with parameters to be used within the body of the procedure. They can also define parameters that return an output value.
```

Comment (caupolicandiaz) [inline content/sql/concepts/procedures/procedures.md:14]: This last sentence feels like it should be better incorporated. Perhaps this content should be added to second sentence - can be defined with parameters to be used with the body of the procedure or to return a value.

File: content/sql/concepts/procedures/procedures.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+The syntax for creating a procedure varies depending upon the type of database management system (DBMS) being used. Below is an example procedure defined and executed with [MySQL](https://www.mysql.com/); it accepts two parameters and returns an output value.
```
Comment (caupolicandiaz) [inline content/sql/concepts/procedures/procedures.md:16]:
```suggestion
The syntax for creating a procedure varies depending upon the type of database management system (DBMS) being used. Below is an example procedure defined and executed with [MySQL](https://www.mysql.com/) that accepts two parameters and returns an output value.
```

**Review by caupolicandiaz — COMMENTED** [review]
a couple of small suggested edits

**Review by yangc95 — APPROVED** [review]

**Comment by SSwiniarski** [comment]
Done with the new comments, @caupolicandiaz, @yangc95.

**Review by caupolicandiaz — APPROVED** [review]

**Comment by yangc95** [comment]
Whoop whoop @SSwiniarski live here: https://www.codecademy.com/resources/docs/sql/procedures

Thanks ya'll 😄 I added some extra spacing to the code block and merged [3731725](https://github.com/Codecademy/docs/commit/3731725f31baa6d117f0b383be19111aaee90d85)

cc: @caupolicandiaz

**Pull request merged**